### PR TITLE
update role error message

### PR DESCRIPTION
**Pull request by muhammed-mamun**
###Summary:

    Fixed and updated error messages in the role-related code to enhance clarity and accuracy.

###Changes:

    Enhanced error messages for role-related validation, making them more descriptive and user-friendly.
    Included minor logic improvements and bug fixes in role-related functionality.

###Related Issues:

    Resolves https://github.com/apache/gravitino/issues/5832

### CODE REVIEW

**Comment by justinmclean** [comment]
The Grvitino CLi mentioned in #5832 is the Java CLI not the python code. However this may still be a good improvement, it just doesn;t fix #5832.

File: clients/client-python/gravitino/dto/requests/catalog_create_request.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def validate(self):
             IllegalArgumentException if name or type are not set.
         """
         if not self._name:
-            raise ValueError('"name" field is required and cannot be empty')
+            raise ValueError('The catalog name is required. Please provide a valid name when creating the catalog.')
```
Comment (tengqm) [inline clients/client-python/gravitino/dto/requests/catalog_create_request.py:66]:
```suggestion
            raise ValueError('The catalog name is required. Please provide a valid name when creating a catalog.')
```

File: clients/client-python/gravitino/dto/requests/schema_create_request.py [hunk 2]
```diff
@@ -42,4 +42,4 @@ def __init__(
 
     def validate(self):
         if not self._name:
-            raise ValueError('"name" field is required and cannot be empty')
+            raise ValueError('"schema name" field is required and cannot be empty')
```
Comment (tengqm) [inline clients/client-python/gravitino/dto/requests/schema_create_request.py:45]: There is no field name "schema name" ... the revised version is confusing.

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "gravitino",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
```
Comment (tengqm) [inline package-lock.json:5]: This file is committed by accident?

**Comment by justinmclean** [comment]
The build failed as the python code was not formatted as expected. It would help if you ran pylint.

**Pull request closed without merging**